src/components/top-tab-bar/TopTabBar.tsx

```diff
@@ -143,11 +143,7 @@ export const TopTabBar = ({ state, descriptors, navigation, position }) => {
               style={styles.tab}
               testID={options.tabBarTestID}
             >
-              {Icon && (
-                <Animated.View style={{ opacity }}>
-                  <Icon />
-                </Animated.View>
-              )}
+              <Animated.View style={{ opacity }}>{icon}</Animated.View>
```

Comment:
Could have sworn that I tried this. Nice

Reply:
the is it a function, is it a component, is it an element had me tripping for a while :D